src/main/resources/global/90-init-alfresco.sh

```diff
@@ -23,6 +34,33 @@ if [[ $SOLR_SSL == none ]] && [[ $ALFRESCO_VERSION != "5.0"* ]] && [[ $ALFRESCO_
   xmlstarlet edit --inplace --delete "/Server/Service[@name='Catalina']/Connector[@SSLEnabled='true' and @port='\${TOMCAT_PORT_SSL}']" $TOMCAT_SERVER_FILE
 fi
 
+# initialiaze variables for keystore location
+DIR_KEYSTORE=${DIR_KEYSTORE:-'/opt/alfresco/keystore'}
+TOMCAT_SSL_KEYSTORE="$DIR_KEYSTORE""/ssl.keystore"
+TOMCAT_SSL_TRUSTSTORE="$DIR_KEYSTORE""/ssl.truststore"
+KEYSTORE_PROPERTIES_FILE="$DIR_KEYSTORE""/ssl-keystore-passwords.properties"
+TRUSTSTORE_PROPERTIES_FILE="$DIR_KEYSTORE""/ssl-truststore-passwords.properties"
+# load custom password variables if custom stores are used
+if [[ "$CUSTOM_KEYSTORES" == true ]]; then
+    TOMCAT_SSL_KEYSTORE_PASSWORD="$(getProperty "$KEYSTORE_PROPERTIES_FILE" "keystore.password")"
+    TOMCAT_SSL_TRUSTSTORE_PASSWORD="$(getProperty "$TRUSTSTORE_PROPERTIES_FILE" "keystore.password")"
+fi
+# write settings as java opts to use for system property substitution in the server.xml
+setJavaOption "-DTOMCAT_SSL_KEYSTORE=$TOMCAT_SSL_KEYSTORE"
+setJavaOption '-DTOMCAT_SSL_KEYSTORE_PASSWORD='${TOMCAT_SSL_KEYSTORE_PASSWORD:-'kT9X6oe68t'}
+setJavaOption "-DTOMCAT_SSL_TRUSTSTORE=$TOMCAT_SSL_TRUSTSTORE"
+setJavaOption '-DTOMCAT_SSL_TRUSTSTORE_PASSWORD='${TOMCAT_SSL_TRUSTSTORE_PASSWORD:-'kT9X6oe68t'}
```

Comment:
Must be quoted in case the password contains spaces
```suggestion
setJavaOption '-DTOMCAT_SSL_TRUSTSTORE_PASSWORD='"${TOMCAT_SSL_TRUSTSTORE_PASSWORD:-'kT9X6oe68t'}"
```